CHANGES.md

```diff
@@ -1,3 +1,11 @@
+## 0.2.8.0
+
+ * Add `Eq1/2`, `Show1/2`, `Read1` instances with `base-4.9
+
+ * `Eq (HashSet a)` doesn't require `Hashable a` anymore, only `Eq a`.
```

Comment:
This is a bit risky because if we ever need the `Hashable` constraint in the future in a new implementation of `Eq`, adding it is a breaking change. Perhaps just leave as is?